src/init.cpp

```diff
@@ -1122,6 +1123,8 @@ bool AppInitParameterInteraction()
 
     nMaxTipAge = gArgs.GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
 
+    SetStartOfScrypt2Period(gArgs.GetArg("-scrypt2peeriod", 0));
```

Comment:
Is it ok for mainnet?

Reply:
it will not change until the value is higher than the start of maintenance period
if someone changes this parameter it will only affect his local node

Reply:
ok